docs/catalog/actions/deployment-restart.rst

```diff
@@ -0,0 +1,8 @@
+Restart Based on prometheus alert
+########################
+
+These actions are useful for automatic restart of deployment based on certain prometheus alert.
```

Comment:
Can you emphasize that we restart the deployment by the configured name, and not necessarily the deployment related to the alert